apps/frontend/app/(website)/studio/features/FirstLoginExperience/config.tsx

```diff
@@ -29,12 +29,12 @@ const steps = [
   {
     element: "output-panel",
     description:
-      "The output panel dynamically reflects changes made in the 'Before' or codemod editor. As users edit the code or the codemod script, the output panel updates in real-time to show the transformed code. This immediate feedback loop helps users quickly identify and correct any issues in their codemod logic.",
+      "<p>The output panel shows how the codemod in the editor affects the 'Before' code snippet in real-time.</p><p>A correct output should be identical to the 'After' code snippet, without introducing false positives or negatives. This immediate feedback loop helps users quickly identify and correct any issues in their codemod logic.</p>",
   },
   {
     name: "Feedback",
     description:
-      "Any feedback? Create an issue in our GitHub repo - <a style='color: blue' href='https://github.com/codemod-com/codemod/'>https://github.com/codemod-com/codemod/</a> ",
+      "Any feedback? <a style='color: blue' href='https://github.com/codemod-com/codemod/'>Create an issue in our GitHub repo</a>",
```

Comment:
Can come as a follow up, Save users one click by sending them to a create issue page